app/vacancy_sources/vacancy_source/source/my_new_term.rb

```diff
@@ -101,19 +101,25 @@ def find_schools(item)
   end
 
   def job_roles_for(item)
-    role = item["jobRole"]&.strip
-    return [] if role.blank?
-
-    # Translate legacy senior/middle leader into all the granular roles split from them
-    return Vacancy::SENIOR_LEADER_JOB_ROLES if role.include? "senior_leader"
-    return Vacancy::MIDDLE_LEADER_JOB_ROLES if role.include? "middle_leader"
-
-    Array.wrap(role.gsub("deputy_headteacher_principal", "deputy_headteacher")
-                   .gsub("assistant_headteacher_principal", "assistant_headteacher")
-                   .gsub("headteacher_principal", "headteacher")
-                   .gsub(/head_of_year_or_phase|head_of_year/, "head_of_year_or_phase")
-                   .gsub(/learning_support|other_support|science_technician/, "education_support")
-                   .gsub(/\s+/, ""))
+    roles = item["jobRole"]&.strip&.split(",")
+    return [] if roles.blank?
+
+    roles.flat_map do |role|
+      # Translate legacy senior/middle leader into all the granular roles split from them
+
+      if role == "senior_leader"
+        Vacancy::SENIOR_LEADER_JOB_ROLES
+      elsif role == "middle_leader"
+        Vacancy::MIDDLE_LEADER_JOB_ROLES
+      else
+        Array.wrap(role.gsub("deputy_headteacher_principal", "deputy_headteacher")
+                       .gsub("assistant_headteacher_principal", "assistant_headteacher")
+                       .gsub("headteacher_principal", "headteacher")
+                       .gsub(/head_of_year_or_phase|head_of_year/, "head_of_year_or_phase")
+                       .gsub(/learning_support|other_support|science_technician/, "education_support")
```

Comment:
We have a role called `other_support` now, so I guess we don't want to map that to `education_support`. Maybe `science_technician` would be better mapped to `other_support` too?